connector-sdk/jackson-datatype-feel/src/main/java/io/camunda/connector/feel/jackson/FeelFunctionDeserializer.java

```diff
@@ -71,10 +71,13 @@ protected Function<IN, OUT> doDeserialize(
 
   @Override
   public JsonDeserializer<?> createContextual(DeserializationContext ctxt, BeanProperty property) {
-    if (property.getType().containedTypeCount() == 2) {
-      var outputType = property.getType().containedType(1);
-      return new FeelFunctionDeserializer<>(outputType, feelEngineWrapper);
+    if (property != null) {
+      if (property.getType().containedTypeCount() == 2) {
```

Comment:
Just curious, when can this be null?

Reply:
Its apparently null when I just want to convert it to an object like `String` and not as part of encapsulating object as a property.